src/main/java/org/sagebionetworks/web/shared/provenance/ActivityGraphNode.java

```diff
@@ -8,17 +8,19 @@
 	private String id;
 	private String activityId;
 	private String activityName;
+	private String activityDescription;
 	private ActivityType type;
 	private String modifiedBy;
 	private Date modifiedOn;
 
 	public ActivityGraphNode() {}
 
-	public ActivityGraphNode(String id, String activityId, String activityName, ActivityType type, String modifiedBy, Date modifiedOn, Boolean startingNode) {
+	public ActivityGraphNode(String id, String activityId, String activityName, String activityDescription, ActivityType type, String modifiedBy, Date modifiedOn, Boolean startingNode) {
```

Comment:
Not entirely sure why all of these serializable graph node objects exist (why we do not just reference the EntityHeader or Activity), but they're still in the codebase.